conans/test/utils/tools.py

```diff
@@ -718,6 +719,15 @@ def assert_listed_binary(self, requires, build=False, test=False):
             else:
                 raise AssertionError(f"Cant find {r}-{kind} in {reqs}")
 
+    def assert_in_file_contents(self, filename, contents: List[str], path=None):
```

Comment:
I looked around (admittedly not _too_ hard) and saw no utility that implemented something like this, and it feels like moving this common check here would be ok